modules/layers/src/column-layer/column-layer.js

```diff
@@ -91,52 +102,75 @@ export default class ColumnLayer extends Layer {
     super.updateState({props, oldProps, changeFlags});
     if (props.fp64 !== oldProps.fp64 || props.diskResolution !== oldProps.diskResolution) {
       const {gl} = this.context;
-      if (this.state.model) {
-        this.state.model.delete();
+      if (this.state.models) {
+        this.state.models.forEach(model => model.delete());
       }
-      this.setState({model: this._getModel(gl)});
+      this.setState(this._getModels(gl));
       this.getAttributeManager().invalidateAll();
     }
-
     if (props.vertices !== oldProps.vertices) {
       this._updateVertices(props.vertices);
     }
   }
 
-  getGeometry(diskResolution) {
-    return new CylinderGeometry({
+  getGeometry(diskResolution, outline = false) {
```

Comment:
Avoid using boolean arguments. It is usually not great for readability.
You can do something like:

```js
const FILL_MODE = 1;
const OUTLINE_MODE = 2;

getGeometry(diskResolution, mode = FILL_MODE) {
  ...
}
```